examples/iotivity_examples/README.md

```diff
@@ -0,0 +1,326 @@
+/sys/bus/usb/devices/2-1.4: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800004321', tty(s): ttyACM1
+```
+We will use Serial Numbers in order to identify the designed node during the compilation phase.
+
+Now, we compile the server
+```
+$ make flash BOARD=samr21-xpro SERIAL=server_node_serial
+```
+then we open the serial connection
+```
+$ make term BOARD=samr21-xpro SERIAL=server_node_serial
+```
+The server starts and it is waiting for incoming requests.
+
+Now, we open a new terminal window, go to `/examples/iotivity-examples/client_switch` and type
+```
+$ make flash BOARD=samr21-xpro SERIAL=client_node_serial
+$ make term BOARD=samr21-xpro SERIAL=client_node_serial
+```
+Client performs the discovery phase. Once it is completed, client registers as an Observer of the resource, then it switches on its LED.
+Client is now ready to send a PUT request when the User Button is pressed. The server LED will change the status when the button is pressed. Terminal outputs are similar to outputs in previous examples.
+##<a name="l2n_comm"></a>Linux-to-Nodes communications
+In this scenario, we will deploy an IoTivity server on a RIOT node and the IoTivity client will run on a Linux machine. This architecture requires the "enhanced" version of the Border Router [BR_FW](br_fw). It requires two SAMR21-XPRO nodes or similar.
+
+###<a name="l2n_pre"></a>Preliminary step
+Connect your nodes, go to `/examples/iotivity-examples/server` and check the list of USB-connected nodes by typing:
+```
+$ make list-ttys
+```
+The output will be similar to
+```
+/sys/bus/usb/devices/2-1.3: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800001234', tty(s): ttyACM0
+/sys/bus/usb/devices/2-1.4: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800004321', tty(s): ttyACM1
+```
+We will use Serial Numbers in order to identify the designed node during the compilation phase.
+
+###<a name="l2n_br"></a>Start the Border Router
+Step 1) Open a terminal window in `/example/iotivity-examples/br_fw/` and type
+```
+$ make flash BOARD=samr21-xpro SERIAL=br_node_serial
+```
+Step 2) Once the flashing is finished, we have to open a network interface. Type
+```
+$  PORT=/dev/ttyACM0 make term
+```
+Step 3) We have to complete the routing strategy on the BR, so in the RIOT shell type
+```
+> ifconfig 7 add 2001:db8::2
+> fibroute add :: via 2001:db8::1 dev 7
+```
+Now the BR is ready. The network is configured as follow:
+- Serial Port: **/dev/ttyACM0**
+- Interface: **tap0**
+- Address Prefix: **2001:db8::\64**
+- Routing on tap0 of multicast packet with destination **ff03::158**
+
+It is possible to configure the network with different parameters by invoking directly the initialization script instead of executing the Step 2:
+```
+$ make host-tools
+$ sudo ./start_network_mcast.sh <serial-port> <tap-device> <IPv6-prefix> <IPv6-mcast>
+```
+then run the Step 3 with the proper changes.
```

Comment:
Does this mean that one has to manually configure the routes on the BR ? Could this be done automatically with the `start_network_mcast.sh` script ?